apps/docs-website/docs/performance-considerations.mdx

```diff
@@ -7,42 +7,49 @@ import TabItem from "@theme/TabItem";
 
 # ⚡ Performance tips
 
-Depending on the device you are operating on, in-editor performance can be a concern with Sheriff. <br />
+Depending on the device you are operating on, in-editor performance can be a concern with Sheriff.<br />
 `@typescript-eslint` can be particularly taxing on the system, so, some performance considerations are in order.
 
 ## Rules performance benchmarking
 
-The currently known slowest rules in Sheriff are the ones of `@typescript-eslint` that [requires type information](https://typescript-eslint.io/getting-started/typed-linting) to work.
+The slowest rules in Sheriff are the ones that [require type information](https://typescript-eslint.io/getting-started/typed-linting) to work.<br />
+typescript-eslint maintains a list [list of type-aware rules](https://typescript-eslint.io/rules/?=typeInformation), which can help track them down.
 
-[List of types-aware rules](https://typescript-eslint.io/rules/?=typeInformation).
-
-You can benchmark rules performance by yourself by running the following command in the terminal:
+You can benchmark rule performance yourself by running the following in the terminal:
 
 ```bash npm2yarn
-TIMING=1 npm run eslint
+TIMING=1 npx eslint
```

Comment:
👎

Reply:
`npx` gets translated just as well as `npm run` with `npm-to-yarn` v3 (included in Docusaurus 3.6), and it otherwise should be `npm exec` anyway.